### ISSUE TITLE
On AS Electric Eel (2022.1.1) Canary 8 and AS Dolphin (2021.3.1) Beta 5 any .kt files with false positive FinalNewline

### ISSUE THREAD

**Issue by rovkinmax** [detekt/detekt-intellij-plugin#207]
I use the latest Detekt plugin version 1.21.0-RC2
In detekt.yml the rule NewLineAtEndOfFile is disabled

**Comment by arturbosch** [comment]
> I use the latest Detekt plugin version 1.21.0-RC2 In detekt.yml the rule NewLineAtEndOfFile is disabled

Hi, the IntelliJ plugin bundles the `detekt-formatting` (KtLint wrapper) plugin by default.
If you have the formatting option enabled you need to disable the KtLint
```yaml
formatting:
  FinalNewline:
    active: false
```
in addition to the detekt native one.

Does this solve your issue?

**Comment by rovkinmax** [comment]
Yes, it solve problem, thanks
